diff --git a/pyproject.toml b/pyproject.toml
index eaf4b3e7..5d6af98a 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -56,10 +56,7 @@ mapbox-earcut = "^1.0.0"
 cloup = ">=0.13,<2.2"
 requests = "^2.26.0"
 dearpygui = { version = "^1.3.1", optional = true }
-skia-pathops = [
-    { version = "^0.7.0", python = "<3.12" },
-    { version = "^0.8.0.post1", python = ">=3.12" }
-]
+skia-pathops = ">=0.7.0"
 isosurfaces = "0.1.0"
 srt = "^3.5.0"
 screeninfo = "^0.8"
